### PR TITLE
change geckodriver version to make consistency

### PR DESCRIPTION
**Pull request by ananzh**
###  Issue Resolved:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2771

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10102,7 +10102,15 @@ https-browserify@^1.0.0:
...
-https-proxy-agent@5.0.0, https-proxy-agent@^5.0.0:
+https-proxy-agent@5.0.1:
+  version "5.0.1"
+  resolved "https://registry.yarnpkg.com/https-proxy-agent/-/https-proxy-agent-5.0.1.tgz#c59ef224a04fe8b754f3db0063a25ea30d0005d6"
+  integrity sha512-dFcAjpTQFgoLMzC2VwU+C/CbS7uRL0lWmxDITmqm7C+7F0Odmj6s9l6alZc6AELXhrnggM2CeWSXHGOdX2YtwA==
+  dependencies:
+    agent-base "6"
+    debug "4"
+
+https-proxy-agent@^5.0.0:
```
Comment (AMoo-Miki) [inline yarn.lock:10113]: Can u plz force these to combine?

Comment (ananzh) [reply yarn.lock:10113]: sure

**Review by AMoo-Miki — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -53,8 +53,9 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Multi DataSource] Address UX comments on index pattern management stack ([#2611](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2611))
 - [Multi DataSource] Apply get indices error handling in step index pattern ([#2652](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2652))
 - [Vis Builder] Last Updated Timestamp for visbuilder savedobject is getting Generated ([#2628](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2628))
-- Removed Leftover X Pack references ([#2638](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2638)) 
+- Removed Leftover X Pack references ([#2638](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2638))
 - Removes Add Integration button ([#2723](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2723))
+- Change geckodriver version to make consistency ([#2772](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2772))
```
Comment (joshuarrrr) [inline CHANGELOG.md:58]: Probably a tossup, but I think this goes under security.

File: yarn.lock [hunk 3]
```diff
@@ -9071,15 +9071,15 @@ gauge@~2.7.3:
     strip-ansi "^3.0.1"
     wide-align "^1.1.0"
 
-geckodriver@^3.0.1:
-  version "3.0.2"
-  resolved "https://registry.yarnpkg.com/geckodriver/-/geckodriver-3.0.2.tgz#6bd69166a24859c5edbc6ece9868339378b6c97b"
-  integrity sha512-GHOQzQnTeZOJdcdEXLuzmcRwkbHuei1VivXkn2BLyleKiT6lTvl0T7vm+d0wvr/EZC7jr0m1u1pBHSfqtuFuNQ==
+geckodriver@^3.0.2:
+  version "3.2.0"
```
Comment (joshuarrrr) [inline yarn.lock:9075]: Is this intentional? `^3.0.2` resolves to `3.2.0`?

Comment (ananzh) [reply yarn.lock:9075]: yes, ^3.0.2 will fetch the latest patch version. though we only require 3.0.2 to solve the cve, there is no breaking changes.

File: CHANGELOG.md (hunk 2 above)
Comment (ananzh) [reply CHANGELOG.md:58]: This is the PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2397 that solves CVE. It is merged before we start updating CHANGLOG. This PR is just for consistency purpose which is brought in by a previous PR. Here are more details:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2771

Comment (ananzh) [reply CHANGELOG.md:58]: Should we put it in security or bug? @joshuarrrr @AMoo-Miki

Comment (joshuarrrr) [reply CHANGELOG.md:58]: lol, I changed my mind - since it's an after the fact cleanup, just put it under "Maintenance".

**Review by joshuarrrr — APPROVED** [review]
I leave it up to you to decide on where to put in in the changelog.

**Pull request merged**